src/getting-started/build-a-sample-app.md

```diff
@@ -70,22 +75,51 @@ impl Component for Model {
     }
 }
 
-fn main() {
-    yew::initialize();
+#[wasm_bindgen(start)]
+pub fn run_app() {
     App::<Model>::new().mount_to_body();
 }
 ```
 {% endcode %}
 
 This template sets up your root `Component`, called `Model` which shows a button that updates itself when you click it. Take special note of `App::<Model>::new().mount_to_body()` inside `main()` which starts your app and mounts it to the page's `<body>` tag. If you would like to start your application with any dynamic properties, you can instead use `App::<Model>::new().mount_to_body_with_props(..)`.
 
+Finally, add an `index.html` file under a new `static` folder for your app.
+
+```bash
+mkdir static
+``` 
+
+{% code title="index.html" %}
+```html
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Yew Sample App</title>
+        <script type="module">
+			import init from "./wasm.js"
+			init()
+		</script>
+    </head>
+    <body></body>
+</html>
+```
+{% endcode %}
+
 ## Run your App!
 
-Using [`cargo-web`](https://github.com/koute/cargo-web) is the quickest way to get up and running. If you haven't already, install the tool with `cargo install cargo-web` and then build and start a development server by running:
+Using [`wasm-pack`](https://rustwasm.github.io/docs/wasm-pack/) is the preferred way to get up and running. If you haven't already, install the tool with `cargo install wasm-pack` and then build and start a development server by running:
 
 ```bash
-cargo web start
+wasm-pack build --target web --out-name wasm --out-dir ./static
 ```
 
-`cargo-web` will automatically add the `wasm32-unknown-unknown` target for you, build your app, and finally make it available at [http://\[::1\]:8000](http://[::1]:8000) by default. Consult `cargo web start --help` for other options.
+`wasm-pack` generates a bundle in the `./static` directory with your app's compiled WebAssembly along with a JavaScript wrapper which can be used to start your application.
```

Comment:
`which can be used` => `which will load your application's WebAssembly binary and run it`